settling_probe.py

```diff
@@ -4,12 +4,12 @@
 # Copyright (C) 2023 Mitko Haralanov <voidtrance@gmail.com>
 #
 # This file may be distributed under the terms of the GNU GPLv3 license.
-import probe
+from .probe import PrinterProbe as PrinterProbe
```

Comment:
If you are not changing the name, you don't need the `as PrinterProbe` part. Without that, it will be imported using it's original name.

Reply:
Opps, sorry about that. You are right.

new squish commit submitted